key-attestation/java-example/README.md

```diff
@@ -21,9 +21,38 @@ Online check
 Offline check
 - `verifyStatementFromJsonWithoutCrlCheck`: Verify given `KeyAttestationResponse` in a JSON file, assuming the last certificate in authority chain is correct ROOT certificate and skipping CRL checks.
 
+# Contributing
```

Comment:
you don't need to repeat this everywhere, the root of the repo is sufficient

Reply:
I see, will delete those extra parts and files